README.md

```diff
@@ -85,6 +85,21 @@ The following are optional for each list:
 
   Specifies that the list requires hostnames be stripped down to the domain boundaries prior to querying the list.  This is required for the [SURBL][3] and [URIBL][4] lists.
 
+* dqs_key
```

Comment:
this is a SpamHaus specific setting. Should it be under the Spamhaus DQS heading?

Reply:
Sounds reasonable, I've moved it there.